Mathlib/Data/Multiset/FinsetOps.lean

```diff
@@ -280,6 +289,12 @@ theorem ndinter_eq_zero_iff_disjoint {s t : Multiset α} : ndinter s t = 0 ↔ D
   rw [← subset_zero]; simp [subset_iff, Disjoint]
 #align multiset.ndinter_eq_zero_iff_disjoint Multiset.ndinter_eq_zero_iff_disjoint
 
+alias ⟨_, Disjoint.ndinter_eq_zero⟩ := ndinter_eq_zero_iff_disjoint
+
+theorem Subset.ndinter_eq_left {s t : Multiset α} (h : s ⊆ t) : s.ndinter t = s := by
```

Comment:
Same, can we have the right verSion?

Reply:
I don't think the right version is meaningfully true? What statement do you want?

Reply:
Ah indeed. But then this lemma is something like `Multiset.filter_true_of_mem h` except that `filter_true_of_mem` doesn't exist

Reply:
It exists for `List` though; I golfed the proof there